src/app/shared/data-elements-list/data-elements-list.component.ts

```diff
@@ -116,9 +116,9 @@ export class DataElementsListComponent implements AfterViewInit {
       )
       .subscribe((data) => {
         this.dataElementRecords = data;
+        this.isLoadingResults = false;
+        this.changeRef.detectChanges();
       });
-
-    this.changeRef.detectChanges();
```

Comment:
The build server has detected a test that failed due to this line being removed. I've been playing with this locally and, although the sorting does work correctly when run in the UI, the `data-elements-list.component.spec.ts` file throws this error:

```
NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'true'. Current value: 'false'.. Find more at https://angular.io/errors/NG0100
```

I'm not exactly sure why this would happen, but I've been experimenting and think if you made these changes then both the test will pass _and_ the UI will sort the table correctly:

1. Add back the `changeRef.detectChanges()` call at the end of this method (_outside_ the `subscribe()` method).
2. Remove the `changeRef.detectChanges()` from inside the `subscribe()` callback.

Please could you action that?